crates/lib3h/src/engine/ghost_engine_wrapper.rs

```diff
@@ -136,7 +141,7 @@ where
 
     /// Process Lib3hClientProtocol message inbox and
     /// output a list of Lib3hServerProtocol messages for Core to handle
-    fn process(&mut self) -> Lib3hProtocolResult<(DidWork, Vec<Lib3hServerProtocol>)> {
+    pub fn process(&mut self) -> Lib3hProtocolResult<(DidWork, Vec<Lib3hServerProtocol>)> {
```

Comment:
So this process used to be `NetworkEngine::process`, now its nothing. Is this supposed to be a ghost * trait?

Reply:
Yah, I think we don't need the trait any more because there will just be one implementation of the engine.  I'll remove the trait.